src/components/MemberSearchPage/ADMINFilter.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
```

Comment:
It's seems to me that all of your filter components have a very similar structure (code duplication?) .Would it be possible to extract the repeating logic in one component?  Eg : const AdminFilter = <MyGenericFilter config={...}/> ;;; const rolesFilter = <MyGenricFilter config={...} /> , ....